### PR TITLE
more fireaxe storage + CE hardsuit temperature buff

### PR DESCRIPTION
**Pull request by Geevies**
I picked suits which would likely need a fireaxe, that being engineering, CE, medical, merc, etc. It makes sense in my head, request changes if you think another one needs it or one doesn't need it.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
Yeah, there's a spriteset for it in belt_mirror.dmi that works for this as-is.

**Comment by Geevies** [comment]
OH SHIT, i just realized that i also changed the CE rig to also be a bit more resistant to fire, i never logged that, FUG

File: html/changelogs/geeves - fireAXE.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - rscadd: "Allows more suits to wear the fireaxe in the suit slot."
```
Comment (Arrow768) [inline html/changelogs/geeves - fireAXE.yml]:
```suggestion
  - tweak: "Allows more suits to wear the fireaxe in the suit slot."
```

File: html/changelogs/geeves - fireAXE.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  - rscadd: "Raised the temperature resistance of the CE hardsuit."
```
Comment (Arrow768) [inline html/changelogs/geeves - fireAXE.yml]:
```suggestion
  - tweak: "Raised the temperature resistance of the CE hardsuit."
```

**Pull request merged**